common/logisticspipes/items/ItemGuideBook.java

```diff
@@ -0,0 +1,35 @@
+package logisticspipes.items;
+
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.util.ActionResult;
+import net.minecraft.util.EnumActionResult;
+import net.minecraft.util.EnumHand;
+import net.minecraft.world.World;
+
+import logisticspipes.network.GuiIDs;
+import logisticspipes.network.PacketHandler;
+import logisticspipes.network.packets.SetCurrentPagePacket;
+import logisticspipes.proxy.MainProxy;
+
+public class ItemGuideBook extends LogisticsItem {
+
+	public ItemGuideBook() {
+		this.setMaxStackSize(1);
+		NBTTagCompound nbt = new NBTTagCompound();
+		nbt.setInteger("page", 1);
+		nbt.setFloat("sliderProgress", 0.0F);
+	}
+
+	public static void setCurrentPage(int page, float sliderProgress, EnumHand hand) {
+		MainProxy.sendPacketToServer(PacketHandler.getPacket(SetCurrentPagePacket.class).setHand(hand).setSliderProgress(sliderProgress).setPage(page));
+	}
+
+	@Override
+	public ActionResult<ItemStack> onItemRightClick(World world, EntityPlayer player, EnumHand hand) {
+		ItemStack itemstack = player.getHeldItem(hand);
+		if (world.isRemote) MainProxy.proxy.openGuideBookGui(hand);
```

Comment:
With the proxy you technically don't need the isRemote check since it won't do anything on the server anyways, but I guess you can keep it

Reply:
I'll let it be then